openedx/core/djangoapps/course_roles/data.py

```diff
@@ -0,0 +1,160 @@
+"""
+Permissions for course roles app.
+
+These are the permissions that can be assigned to a CourseRole which grants access for a course,
+but can also be assigned for org wide course access or instance wide course access.
+They are defined in the database in the course_roles_permission table.
+
+To add a new permission, add a new entry to the CourseRolesPermission enum,
+then and add a new row to the course_roles_permission table in database,
+with a migration.
```

Comment:
You might want to be explicit that they need to create the new data migration in this app, and not in a third party app that they're developing. (At least for now.)